unit10/lab02bee/SpellingBee.java

```diff
@@ -39,12 +59,17 @@ public static String loadFile(String filename) {
     public static void main(String[] args) {
         String[] words = loadFile("words_dropped.txt").split("\n");
         System.out.println("Loaded " + words.length + " words");
-        // TODO solve me!
         // SpellingBee bee = new SpellingBee("ranglty".toCharArray(), 'y');
+        SpellingBee bee = new SpellingBee("ranglty".toCharArray(), 'y');
+        ArrayList 
+        for (int t = 0; t < words.length; t++){
+            if (checkWord(words[t]) == true)
```

Comment:
You have an errant `ArrayList` on line 64, and then since `checkWord` is not a static function, you need to call it on an instance of the object `bee` you defined on line 63.  bee.checkWord